css/navigation.css

```diff
@@ -0,0 +1,60 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css");
+
+
+.bg-color{
+  background-color:#8ca0ff
+}
+
+
+.header1{
+    
+    display:grid;
+    grid-template-columns:1.5fr 1fr 1fr 1fr 
+    
+   }
+.header2{
+    
+    display:grid;
+    grid-template-columns:5fr 2fr 2fr 2fr 
+    
+   }
+
+.header3{
+    
+display:grid;
+grid-template-columns:2.4fr 1fr 1fr 1fr 1fr  }
```

Comment:
Use lists in your navbars . And then do the layout using Flexbox.

Reply:
Using grid isn't good practice ?